aat/src/java/uk/gov/hmcts/ccd/printservice/befta/CasePrintServiceTestAutomationAdapter.java

```diff
@@ -12,4 +16,21 @@ public void doLoadTestData() {
         loader.addCcdRoles();
     }
 
+    @Override
+    public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenarioContext, Object key) {
+        if (key.toString().startsWith("approximately ")) {
+            try {
+                String actualSize = (String)ReflectionUtils.deepGetFieldInObject(scenarioContext, "testData.actualResponse.body.__fileInBody__.size");
+                String expectedSize = key.toString().replace("approximately ", "");
+                int actualValue = Integer.parseInt(actualSize);
+                int expectedValue = Integer.parseInt(expectedSize);
+                if(Math.abs(actualValue-expectedValue)<50)
```

Comment:
Does the coding standard for the repo call for spaces around operators such as `-` and `<`?

Reply:
done